code/modules/psionics/faculties/coercion.dm

```diff
@@ -249,7 +234,7 @@
 	admin_log = FALSE
 
 /datum/psionic_power/coercion/commune/invoke(var/mob/living/user, var/mob/living/target)
-	if((target == user) || user.zone_sel.selecting != BP_MOUTH)
+	if(user.zone_sel.selecting != BP_MOUTH)
```

Comment:
I think we do need this if(target==user) bit.

Reply:
Why?